src/CommunityToolkit.Maui.Core/Views/Alert/Alert.macios.cs

```diff
@@ -6,28 +6,28 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// <summary>
 /// Popup for iOS + MacCatalyst
 /// </summary>
-public class Popup
+public class Alert
 {
 	NSTimer? timer;
 
 	/// <summary>
 	/// Initialize Popup
 	/// </summary>
-	public Popup()
+	public Alert()
 	{
-		PopupView = new PopupView();
+		PopupView = new AlertView();
```

Comment:
Let's rename the property to AlertView as well

Reply:
Another good catch! Done ✅